test-basic/bindingFromParam.js

```diff
@@ -20,9 +20,33 @@ const db = marklogic.createDatabaseClient(testconfig.restWriterConnection);
 const op = marklogic.planBuilder;
 let should = require('should');
 const fs = require("fs");
+const testlib = require("../etc/test-lib");
+let serverVersionGreaterThanEqual11 = false;
 
-describe('binding from param', function () {
-    this.timeout(20000);
+describe('optic-update fromParam tests', function(){
+
+    before(function (done) {
+        const manageClient =
```

Comment:
This is a lot of duplication of the same code - couldn't this all become a function in test-lib, like `requiresMarkLogic11()`?

Reply:
Added findServerConfiguration in test-lib. We will also be able to use it later for tests related to features going in MarkLogic 12